src/guides/resource-records.md

```diff
@@ -0,0 +1,274 @@
+## Handshake Resource Record Guide
+
+Handshake has resource records that represent DNS resource records.
+Some of them map one-to-one with DNS resource record types,
+while others do not. These resource records are included in
+`REGISTER` or `UPDATE` outputs and written to the Urkel tree
+so that proofs can be served to light clients.
+
+Each Handshake name stores a single `Resource` with the maximum size of
+512 bytes.
+
+### Updating Resource Records
+
+To update the resource records associated with a name, the wallet RPC
+command `sendupdate` is used. If the name is in a different wallet
+than the primary wallet, use the RPC command `selectwallet` before
+sending the update. The Node RPC method `verifyresource` can be used
+to check if the resource is valid before attempting to `sendupdate`.
+
+```bash
+$ hsw-cli rpc sendupdate easyhandshake '{"records": []}'
+```
+
+Note that there are no `records` being included in this example.
+When calling `sendupdate` for the first time after winning an
+auction, a `REGISTER` output is created along with a change output
+that includes the difference between the user's bid and the
+second highest bid. Use the Node RPC `getnameresource` to view
+the resources for a name. Newly updated names will not reflect their
+records via DNS until a tree inverval is passed, which happens
+every 36 blocks.
```

Comment:
I *think* this only pertains to light clients. If you run a full node, I believe the new records are available immediately after the UPDATE is confirmed.